### DISCUSSION TITLE
Install depthmap2mask extension in webUI errors

### DISCUSSION THREAD

**Discussion by larsbars** [TheLastBen/fast-stable-diffusion#856] [General]
I get this when trying to install through the UI.  

AssertionError: extension access disabled because of command line flags

It seems this is due to running with --share, which I think is required in order to use the webUI with colab right?  I must be missing something simple ....

**Comment by TheLastBen** [comment]
uncheck the box "use_gradio_server", that will allow to install the extention

**Reply by larsbars** [reply]
Thank you!  I had somehow gotten into the habit of using gradio and thinking that was the only way to connect.

**Reply by larsbars** [reply]
i'm subsequently getting "FileNotFoundError: [Errno 2] No such file or directory: 'extensions/depthmap2mask/scripts/depthmap_for_depth2img.py'" at startup and "FileNotFoundError: [Errno 2] No such file or directory: 'extensions/depthmap2mask/scripts/depthmap_for_depth2img.py'" when trying to generate with the extension.  Extension may be broken huh?
